spec/features/user_visits_homepage_spec.rb

```diff
@@ -9,14 +9,14 @@
 
   scenario "successfully redirects to sign in page" do
     visit root_path
-    click_link 'Sign in'
+    click_button "Sign in"
```

Comment:
Hmm, based on the redesigns that I made, this test shouldn't even be passing (this is based on the previous design). I think you are running your tests based on master, not the branch I am on. Did you pull down my branch locally and create a branch off of there?